### PR TITLE
Completing sentence

### PR DESCRIPTION
**Pull request by singgihbekti**
there is a sentence that doesn't have a verb.

### CODE REVIEW

File: en_ID/pages/internship.md [hunk 1]
```diff
@@ -176,10 +176,18 @@ Note: This not required for a non-technical candidate. A non-technical candidate
 At this fifth stage, we will discuss a little how-to to keeping fork updated. So the goal is how we can get the latest version from the upstream repository and then merge the new changes into your fork.
 
 Note: you can follow this steps below<br />
-**1**. Add the original remote URL repository with this command `git remote add upstream <REPO_URL>` ,the original repository is mostly called `upstream`.
+**1**. Add the original remote URL repository with this command `git remote add upstream <REPO_URL>`, the original repository is mostly called `upstream`.
 **2**. To make sure remote is added, you can check with this command `git remote -v`.
 **3**. Now, to keep `upstream` updated run this command `git fetch upstream`.
 
+This is an example for the code
```
Comment (empeje) [inline en_ID/pages/internship.md:183]: You can use the ``` syntax to make the code multiline here instead of having an inline example.

**Comment by empeje** [comment]
Good job on your first PR @singgihbekti. Could you please Make the PR title and description better, e.g. using title case for Title and use sentence case for the description?

File: en_ID/pages/internship.md [hunk 2]
```diff
@@ -176,10 +176,18 @@ Note: This not required for a non-technical candidate. A non-technical candidate
 At this fifth stage, we will discuss a little how-to to keeping fork updated. So the goal is how we can get the latest version from the upstream repository and then merge the new changes into your fork.
 
 Note: you can follow this steps below<br />
-**1**. Add the original remote URL repository with this command `git remote add upstream <REPO_URL>` ,the original repository is mostly called `upstream`.
+**1**. Add the original remote URL repository with this command `git remote add upstream <REPO_URL>`, the original repository is mostly called `upstream`.
```
Comment (empeje) [inline en_ID/pages/internship.md:179]: It seems these changes carried over changes from the https://github.com/kulkultech/open-source/pull/115. One Pull Request should only contain one purpose.

File: en_ID/pages/internship.md (hunk 1 above)
Comment (singgihbekti) [reply en_ID/pages/internship.md:183]: Okay, thank you @empeje. I revise the code using the multiline format. 
The result can be viewed here: 
https://raw.githack.com/singgihbekti/open-source/feature/gh-pages-sb/en_ID/#!pages/internship.md#Step_5_-_Keeping_fork_updated

**Comment by empeje** [comment]
Although the commit is still mixed up with other PR, I'll merge it. Keep up the good work!

**Pull request merged**